src/workflows-app/components/LeftNavigationPanel.ts

```diff
@@ -0,0 +1,156 @@
+            titleFirst: true,
+            summaryStyle: { padding: '1rem 1rem 1rem 1.5rem' },
+          },
+          [
+            div(
+              {
+                style: { alignItems: 'center', flexDirection: 'column', paddingLeft: '2rem' },
+                role: 'list',
+              },
+              [
+                _.map(([subHeaderKey, subHeaderName]) => {
+                  const isActive = isSubHeaderActive(subHeaderKey);
+                  return h(
+                    Clickable,
+                    {
+                      'aria-label': `${subHeaderKey}-header-button`,
+                      style: { color: isActive ? colors.accent(1.1) : colors.accent(), fontSize: 16 },
+                      onClick: () => setSelectedSubHeader(subHeaderKey),
+                      hover: Style.navList.itemHover(isActive),
+                      'aria-current': isActive,
+                      key: subHeaderKey,
+                    },
+                    [
+                      h(ListItem, {
+                        title: subHeaderName,
+                        style: { borderBottom: `0.5px solid ${colors.dark(0.2)}` },
+                      }),
+                    ]
+                  );
+                }, Object.entries(findAndAddSubheadersMap)),
+                div(
+                  {
+                    style: {
+                      marginRight: '3em',
+                      marginTop: '1.5em',
+                      display: 'flex',
+                      alignItems: 'center',
+                      flex: 'none',
+                      backgroundColor: colors.accent(0.06),
+                      boxShadow: '0 2px 5px 0 rgba(0,0,0,0.35), 0 3px 2px 0 rgba(0,0,0,0.12)',
+                      paddingTop: '0.5em',
+                      paddingBottom: '0.5em',
+                      paddingLeft: '0.75em',
+                      paddingRight: '0.75em',
+                      borderRadius: '15px',
+                    },
+                  },
+                  [
+                    h(
+                      Clickable,
+                      {
+                        role: 'list',
+                        href: `${
+                          getConfig().dockstoreUrlRoot
+                        }/search?_type=workflow&descriptorType=WDL&searchMode=files`,
+                      },
+                      [
+                        div({ style: { fontWeight: 'bold' } }, ['Dockstore  ', icon('export')]),
+                        div([
+                          'Browse WDL workflows in Dockstore, an open platform used by the GA4GH for sharing Docker-based workflows.',
+                        ]),
+                      ]
+                    ),
+                  ]
+                ),
+              ]
+            ),
+          ]
+        ),
+      ]),
+      h(HelpfulLinksBox, {
+        method: null,
+        style: { borderRadius: '20px', margin: '1.35em', bottom: '0', position: 'fixed' },
```

Comment:
It seems the `HelpfulLinksBox` in the panel and the one rendered in Submission Config page now have different UX (rounded vs square edges). It might be better to make it consistent. It might be good to ask Sara and/or Katie to see which style should be followed in both places.
In panel:
![Screenshot 2023-09-11 at 1 25 48 PM](https://github.com/DataBiosphere/terra-ui/assets/16748522/fcf4dda3-9aad-423a-a8a6-c9bb2f3bf9ba)

In Submission Config page:
![Screenshot 2023-09-11 at 1 26 12 PM](https://github.com/DataBiosphere/terra-ui/assets/16748522/b6dca456-e214-4cd3-8758-17f61f7ea3f1)

Reply:
@kpierre13 What was the decision regarding the UX of `HelpfulLinksBox`?

Reply:
After talking to Katie/Lou, this was the final decision for these info boxes:

![image](https://github.com/DataBiosphere/terra-ui/assets/68349264/94f96e37-545d-4888-8b62-b6ade1174aaa)
![image](https://github.com/DataBiosphere/terra-ui/assets/68349264/d38c807c-4f07-4c47-86b7-2943e5bac1df)

Reply:
I made sure that there were no axe errors regarding these style changes.